dev-docs/publisher-api-reference/bidderSettings.md

```diff
@@ -209,7 +215,32 @@ pbjs.bidderSettings = {
 
 In the above example, the AOL bidder will inherit from "standard" adserverTargeting keys, so that you don't have to define the targeting keywords again.
 
-##### 2.3. sendStandardTargeting
+##### 2.3. inverseCpmAdjustment
+
+When using [price floors](dev-docs/modules/floors.html), Prebid attempts to calculate the inverse of `bidCpmAdjustment`, so that the floor values it requests from SSPs take into account how the bid will be adjusted. 
+For example, if the adjustment is `bidCpm * .85` as above, floors are adjusted by `bidFloor * 1 / .85`.
+
+The automatically derived inverse function is correct only when `bidCpmAdjustment` is linear. If it isn't, the inverse should also be provided through `inverseCpmAdjustment`. For example:
```

Comment:
Because we might get some pushback, I'm not sure that being "linear" is the requirement -- y=mx+b is linear, but it's shifted off the origin. Likewise, (cpm - 0.2) * .85 is linear. 

Would suggest we reword this to "is correct only when bidCpmAdjustment function only involves multiplication." Agree?

Reply:
Agree, I was getting ahead of myself since I plan to make it work with linear offsets, but right now it is in fact only correct if it's a pure multiplier.